### DISCUSSION TITLE
[Answer] Yes! Is threre any console manufacturer licensed game supports screen rotation?

### DISCUSSION THREAD

**Discussion by soundsnow** [project64/project64#2343] [Ideas]
### Request box

These are pasted from my another issue, if you have seen some of them, do not reply to me

Licensed Commercial and Homebrew Games
Orientation: Auto, Portrait, Landscape
Screen Rotation: 0, 90°, 180°, 270°
4:3 to 3:4
16:9 to 9:16
21:9 to 9:21

This game Ikaruga is made by a famous video game developer. Treasure released this game for Sega Naomi Arcade Machine and Dreamcast Home Console. Dreamcast version is made for rotating monitor obviously.
To play this game like Arcade version
Open Game: Config - Monitor Type (Vertical 1/Vertical 2)
Vertical 1: Default D-Pad Direction
Vertical 2: Follow Screen Rotation D-Pad Direction
Rotate your monitor
If you don't have a Rotating Monitor, Emulator Settings - Screen Rotation (0, 90°, 180°, 270°)

Portrait Orientation and Screen Rotation are both required

Screen Rotation supported video game devices
Wii U
Lynx
WonderSwan
DS
DSi
3DS
New 3DS
PSP with 2 (Right) Analog Sticks
PSVita
Switch
Arcade Machine
Rotation Monitor (Home Console & PC)
Wireless Controller (Handheld Console)
Symbian, Windows Phone, iPhone, Android (Cellphone)
Emulator (Software Simulation)

![image58](https://user-images.githubusercontent.com/109885044/219099264-7d4044e4-507e-480c-a1c4-f8ce186cb737.png)
![image61](https://user-images.githubusercontent.com/109885044/219247030-a65484dd-79af-4371-9494-cd91899a8b74.png)
![image62](https://user-images.githubusercontent.com/109885044/219275604-a3c93f7f-185d-434c-a7b5-da6e1fcd0de2.png)

### Reason box

I submit this issue because it seems you'll rerelease new Android version

### Examples

O

**Comment by project64** [comment]
I have no idea what your asking that needs to be changed

**Reply by soundsnow** [reply]
Home Console aspect ratio doesn't allow you to play vertical shooters like Arcade Machine
As far as I know there's a vertical shooter for Aleck64
A N64 related game - Variant Schwanzer
https://www.system16.com/hardware.php?id=816
Android phone requires Portrait Orientation and Screen Rotation 90°, 270° to play Nintendo Licensed and Homebrew vertical shooters
PC rotating monitor Fullscreen 4:3,16:9 clockwise/counterclockwise 90° 3:4,9:16
Ikaruga Dreamcast version is a good example
![image64](https://user-images.githubusercontent.com/109885044/219524486-5d169fec-5f7a-440d-b1dc-9e6151a3c550.png)
PSP (Dual Analog) and PSVita
![image65](https://user-images.githubusercontent.com/109885044/219526537-07786ea0-c784-4fb9-8d3f-75246ac1ab3e.png)
